components/next_steps_view/steps/invite_members_step/invite_members_step.tsx

```diff
@@ -32,6 +36,7 @@ type Props = StepComponentProps & {
         sendEmailInvitesToTeamGracefully: (teamId: string, emails: string[]) => Promise<{ data: TeamInviteWithError[]; error: ServerError }>;
         regenerateTeamInviteId: (teamId: string) => void;
         getSubscriptionStats: () => void;
+        openModal?: (modalData: { modalId: string; dialogType: any; dialogProps?: any }) => void;
```

Comment:
Given that we bind this prop for `openModal` any special reason this is marked as optional?